doc/user_guide/psyke.rst

```diff
@@ -241,26 +241,26 @@ to insert the extract region. As shown below, all children of an
       0: Loop[type='dofs',field_space='any_space_1',it_space='dofs',
               upper_bound='ndofs']
           ...
-	  Schedule[]
+      Schedule[]
               0: BuiltIn setval_c(f5,0.0)
       1: Loop[type='dofs',field_space='any_space_1',it_space='dofs',
               upper_bound='ndofs']
-	  ...
-	  Schedule[]
+      ...
```

Comment:
This line is not indented correctly - it should line up with the `1: ` above.

Reply:
I realised I messed up indenting when replacing tabs with spaces in this file. Hopefully corrected now.